molecule/tests/roles/haproxy/variables/haproxy.cfg.yml

```diff
@@ -0,0 +1,163 @@
+---
+# 🚀 These tasks aim to test the "ansible.builtin.lineinfile" task
+# 🎯 The objective is to ensure that the lines are correctly replaced
+
+# 📂 Ensure tmp directory exists
+- name: Molecule.tests.roles.confd.variables.haproxy.cfg | Ensure tmp directory exists
+  run_once: true
+  delegate_to: localhost
+  ansible.builtin.file:
+    path: "./tmp"
+    state: directory
+
+# 🔄 Define a dummy template file
+- name: Molecule.tests.roles.confd.variables.haproxy.cfg | Set template file test data
+  run_once: true
+  ansible.builtin.set_fact:
+    haproxy_listen_port:
+      stats: 5000
+      master: 5001
+      replicas: 5002
+      replicas_sync: 5003
+      replicas_async: 5004
```

Comment:
@vitabaks,

Would it be more appropriate to do this ?

```yaml
# 🔄 Specify a placeholder template file
- name: Molecule.tests.roles.confd.variables.haproxy.cfg | Define test data for the template file
  run_once: true
  ansible.builtin.set_fact:
    haproxy_listen_port:
      master: 5000
      replicas: 5001
      replicas_sync: 5002
      replicas_async: 5003
      stats: 7000
```

Reply:
Yes

See `haproxy_listen_port` variable https://github.com/vitabaks/postgresql_cluster/blob/master/vars/main.yml#L27

Reply:
Done